ini/stm32f1.ini

```diff
@@ -374,3 +374,43 @@ extra_scripts        = ${stm32_variant.extra_scripts}
 platform      = ${common_stm32.platform}
 extends       = env:chitu_f103
 build_flags   = ${env:chitu_f103.build_flags} -DCHITU_V5_Z_MIN_BUGFIX
+
+#
+# Zonestar ZM3E2 V1.0 / ZM3E4 V1.0 / ZM3E4 V2.0
+#
+#  STM32F103RC_ZM3E2_USB ........... RCT6 with 256K
+#  STM32F103VC_ZM3E4_USB ........... VCT6 with 256K
+#  STM32F103VE_ZM3E4V2_USB ......... VET6 with 512K
+#
+[ZONESTAR_ZM3E]
+platform                  = ${common_stm32.platform}
+extends                   = stm32_variant
+platform_packages         = ${stm_flash_drive.platform_packages}
+board_upload.offset_address = 0x08005000
+board_build.offset        = 0x5000
+board_upload.maximum_size = 237568
+extra_scripts             = ${stm32_variant.extra_scripts}
+build_flags               = ${common_stm32.build_flags}
+  -DSS_TIMER=4 -DTIMER_SERVO=TIM5 -DUSE_USB_FS -DUSBD_IRQ_PRIO=5 -DUSBD_IRQ_SUBPRIO=6 -DUSBD_USE_CDC_MSC
+build_unflags             = ${stm32_variant.build_unflags} -DUSBD_USE_CDC
+
+[env:STM32F103RC_ZM3E2_USB]
+platform             = ${ZONESTAR_ZM3E.platform}
+extends              = ZONESTAR_ZM3E
+board                = genericSTM32F103RC
+board_build.variant  = MARLIN_F103Rx
+
+[env:STM32F103VC_ZM3E4_USB]
+platform             = ${ZONESTAR_ZM3E.platform}
+extends              = ZONESTAR_ZM3E
+board                = genericSTM32F103VC
+board_build.variant  = MARLIN_F103Vx
+build_flags          = ${ZONESTAR_ZM3E.build_flags} -DTONE_TIMER=1 -DTONE_CHANNEL=2
```

Comment:
these defines are for maple only... STM32 use -DTIMER_TONE ... no channel then just a pin

Reply:
I see. Just being contrarian, are they? What's the other one… "`-DTHE_CHANNEL_FOR_THE_TONE_TIMER`" ?

Reply:
I see that `pins_BTT_SKR_MINI_E3_V2_0.h` also contains …
```cpp
#define BOARD_CUSTOM_BUILD_FLAGS -DTONE_CHANNEL=4 -DTONE_TIMER=4
```
I assume that this also only compatible with Maple, and is ignored on the non-Maple platforms.